Mopups/Mopups.Maui/Platforms/Android/Impl/AndroidMopups.cs

```diff
@@ -40,13 +38,32 @@ public Task AddAsync(PopupPage page)
     {
         try
         {
-            var decoreView = DecoreView;
+            HandleAccessibility(ImportantForAccessibility.NoHideDescendants);
+            DisableFocusableInTouchMode((Application.Current?.MainPage?.Handler?.PlatformView as Android.Views.View)?.Parent);
 
             page.Parent = MauiApplication.Current.Application.Windows[0].Content as Element;
             var AndroidNativeView = IPopupPlatform.GetOrCreateHandler<PopupPageHandler>(page).PlatformView as Android.Views.View;
-            decoreView?.AddView(AndroidNativeView);
+            DecoreView?.AddView(AndroidNativeView);
 
             return PostAsync(AndroidNativeView);
+
+            static void DisableFocusableInTouchMode(IViewParent? parent)
+            {
+                var view = parent;
+                string className = $"{view?.GetType().Name}";
+
+                while (!className.Contains("PlatformRenderer") && view != null)
+                {
+                    view = view.Parent;
+                    className = $"{view?.GetType().Name}";
```

Comment:
Is the name "PlatformRenderer" something under our control? or MAUI's? Just getting a quick grip with how this works